oshc/main/admin.py

```diff
@@ -10,5 +10,9 @@ class chatSessionAdmin(admin.ModelAdmin):
     list_display = ('title', 'start_date')
 
 
+class contestAdmin(admin.ModelAdmin):
+    list_display = ('name', 'link', 'description', 'start_date', 'end_date', 'approved')
```

Comment:
This won't enable inline edit. Admin still need to go inside the event and say approve/disapprove and save.